### PR TITLE
added a Blade of Wrath proc tracker and added more spell links in my suggestions

### PR DESCRIPTION
**Pull request by CollCrom**

### CODE REVIEW

File: src/Parser/RetributionPaladin/Modules/PaladinCore/BoWProcTracker.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		const spellId = event.ability.guid;
+		if(spellId !== SPELLS.BLADE_OF_WRATH_PROC.id){
+			return;
+		}
+		this.overwrittenBoWProcs++;
+		this.totalBoWProcs++;
+	}
+
+	suggestions(when) {
+		const missedProcsPercent = this.overwrittenBoWProcs / this.totalBoWProcs;
```
Comment (anom0ly) [inline src/Parser/RetributionPaladin/Modules/PaladinCore/BoWProcTracker.js]: Just a question, is there ever a time where this buff could expire without being used? If so you may want to include the `removebuff` trigger to account for that.

File: src/Parser/RetributionPaladin/Modules/PaladinCore/BoWProcTracker.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	on_initialized(){
+		this.active = this.combatants.selected.hasTalent(SPELLS.BLADE_OF_WRATH_TALENT.id);
+	}
+	
+	on_byPlayer_applybuff(event){
+		const spellId = event.ability.guid;
+		if(spellId !== SPELLS.BLADE_OF_WRATH_PROC.id){
+			return;
+		}
+		this.totalBoWProcs++;
```
Comment (anom0ly) [inline src/Parser/RetributionPaladin/Modules/PaladinCore/BoWProcTracker.js]: For code style, please use the format `this.totalBoWProcs += 1;`

Just to be consistent with the rest of the application's style.

File: src/Parser/RetributionPaladin/Modules/PaladinCore/BoWProcTracker.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	}
+
+	on_byPlayer_refreshbuff(event){
+		const spellId = event.ability.guid;
+		if(spellId !== SPELLS.BLADE_OF_WRATH_PROC.id){
+			return;
+		}
+		this.overwrittenBoWProcs++;
+		this.totalBoWProcs++;
```
Comment (anom0ly) [inline src/Parser/RetributionPaladin/Modules/PaladinCore/BoWProcTracker.js]: See above for the updated code style to handle incremental increases in variables.

File: src/Parser/RetributionPaladin/Modules/Features/Judgment.js [hunk 4]
```diff
@@ -61,7 +62,7 @@ class Judgment extends Module {
 		const unbuffedJudgmentPercentage = this.spenderOutsideJudgment / this.totalSpender;
 		when(unbuffedJudgmentPercentage).isGreaterThan(0.05)
 			.addSuggestion((suggest,actual,recommended) => {
-				return suggest('You\'re spending Holy Power outisde of the Judgment debuff Window too much. Only spending while the enemy has Judgment on them is very important.')
+				return suggest(<span>You're spending Holy Power outisde of the <SpellLink id={SPELLS.JUDGMENT_CAST.id} /> debuff Window too much. Only spending while the enemy has the debuff on them is very important.</span>)
```
Comment (anom0ly) [inline src/Parser/RetributionPaladin/Modules/Features/Judgment.js]: Just need to clean up the language here. I get the underlying point but potentially reword this to stay something like:
`You're spending Holy Power outside of the Judgment debuff. It is optimal to only spend Holy Power while the enemy is debuffed with Judgement.`

File: src/Parser/RetributionPaladin/Modules/Features/CastEfficiency.js [hunk 5]
```diff
@@ -79,7 +82,7 @@ class CastEfficiency extends CoreCastEfficiency {
       getCooldown: haste => 12 / (1 + haste),
       isActive: combatant => combatant.hasBuff(SPELLS.RET_PALADIN_T20_2SET_BONUS_BUFF.id),
       recommendedCastEfficiency: 0.95,
-      extraSuggestion: 'With tier 20 2 peice it is even more imporant to use Judgment on cooldown to keep up the buff',
+      extraSuggestion: 'With tier 20 2 peice it is even more imporant to use <SpellLink id={SPELLS.JUDGMENT_CAST.id} /> on cooldown to keep up the buff',
```
Comment (anom0ly) [inline src/Parser/RetributionPaladin/Modules/Features/CastEfficiency.js]: Just check the spelling on this one.

File: src/Parser/RetributionPaladin/Modules/PaladinCore/BoWProcTracker.js (hunk 1 above)
Comment (CollCrom) [reply src/Parser/RetributionPaladin/Modules/PaladinCore/BoWProcTracker.js]: I am currently not playing WoW anymore so I will have someone check for me but I am pretty sure it just instantly brings the spell off cooldown. The only time the buff can expire is if you get another proc without using the first one.

**Comment by CollCrom** [comment]
Ahh the english language my greatest enemy

**Comment by anom0ly** [comment]
Looks good! And sorry for the delay in getting this merged. Just ping me in Discord or here if I end up not responding for more than a day.
